CONTRIBUTING.md

```diff
@@ -85,25 +85,28 @@ development with the following:
 
   For Url:
 
-  `http://localhost:5000/`
+  `http://localhost:5000/auth/myusa/callback`
 
   For Redirect uri:
 
   `http://localhost:5000/auth/myusa/callback`
 
-2. Generate a set of keys, `MYUSA_KEY` and `MYUSA_SECRET`, and reference them in
+2. Under `Select the API Scopes that your Application will use`: select `Email
+   Address`.
+
+3. Generate a set of keys, `MYUSA_KEY` and `MYUSA_SECRET`, and reference them in
```

Comment:
Is there some kind of guest/limited access account to the 18f slack? I could see it being annoying for people to have to create a new slack organization to test things. Would it be feasible to enable contributors to have access to a special contributors room under the 18f slack with the ability to add bots? I haven't really messed around with managing a slack organization but this could be just another way to lower the barrier of entry for those who might not have used slack before and want to get into this project quickly. Do you see this as at all feasible?